### PR TITLE
Add persistence entity for database model appointment

### PR DESCRIPTION
**Pull request by aca24**

### CODE REVIEW

**Review by dusancina98 — CHANGES_REQUESTED** [review]
CI pipeline failed during PR

File: MQuince/MQuince.Application/App.cs [hunk 1]
```diff
@@ -1,59 +0,0 @@
...
-        private IPatientRepository GetPatientRepository()
-             => new PatientRepository(_optionsBuilder);
-
-        public IDoctorService GetDoctorService()
-              => new DoctorService(this.GetDoctorRepository());
-
-        private IDoctorRepository GetDoctorRepository()
-             => new DoctorRepository(_optionsBuilder);
-    }
-}
```
Comment (zindovicstefan) [inline MQuince/MQuince.Application/App.cs:59]: This class should not be deleted, application can't work properly without it

**Review by zindovicstefan — CHANGES_REQUESTED** [review]
Potential conflicts in appointment component

**Comment by aca24** [comment]
Can you anyone help me?

pet, 04. dec 2020. 21:03 zindovicstefan <notifications@github.com> je
napisao/la:

**Pull request closed without merging**